docs/pages/concept/web.md

```diff
@@ -0,0 +1,145 @@
+# Web Renderer & Web View
+
+Web rendering is an experimental feature that lets you render websites. 
+Furthermore, you have the flexibility to seamlessly integrate additional components as children and render them within the website. We refer to this process as embedding.
```

Comment:
```suggestion
Furthermore, you can seamlessly integrate additional components as children and render them within the website. We refer to this process as embedding.
```
Avoid unnecessary filler words, You can use Grammarly to detect cases like this.  

Also, I'm not really sure what "integrate additional components as children and render them" means.
- integrate as children? - no idea?
- integrate (...) and render? - What is integrating and what is rendering?